### PR TITLE
Implemented session tab animation

### LINKED ISSUE #328: Session tab animation
## Overview (Required)
- Move the chip frame
- It's very exciting. (I do not know how to implement it 😉 )
https://storage.googleapis.com/spec-host-backup/mio-design%2Fassets%2F1ZL4CiG4FN1MxyTuSqZqU_xX_V5pDakCs%2Fnavtransitions-peer.mp4

### PR DESCRIPTION
**Pull request by NUmeroAndDev**
## Issue
- close #328 

## Overview (Required)
- Implemented session tab animation.
But, RippleEffect is not the same as before. 
It was difficult for me. 😢 

## Links
- https://storage.googleapis.com/spec-host-backup/mio-design%2Fassets%2F1ZL4CiG4FN1MxyTuSqZqU_xX_V5pDakCs%2Fnavtransitions-peer.mp4

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: feature/session/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -4,4 +4,10 @@
     <style name="Widget.App.ChipGroup.SessionFilter" parent="Widget.MaterialComponents.ChipGroup">
         <item name="chipSpacingVertical">8dp</item>
     </style>
+
+    <style name="TabTextAppearance" parent="TextAppearance.App.Body2">
```
Comment (takahirom) [inline feature/session/src/main/res/values/styles.xml]: 👍

File: feature/session/src/main/res/layout/fragment_session_pages.xml [hunk 2]
```diff
@@ -16,13 +16,14 @@
...
             android:layout_height="wrap_content"
-            android:background="?colorPrimary"
             android:tabStripEnabled="false"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent"
-            app:tabIndicatorColor="@android:color/transparent"
-            app:tabRippleColor="@null"
+            app:tabIndicator="@drawable/shape_pill_indicator"
+            app:tabIndicatorGravity="stretch"
```
Comment (takahirom) [inline feature/session/src/main/res/layout/fragment_session_pages.xml]: 🆒

**Comment by takahirom** [comment]
Sorry, I want to change some layout.
Probably circle frame height is 40dp but the design size is 32dp. Can you apply it?
image

**Comment by NUmeroAndDev** [comment]
@takahirom 
> Probably circle frame height is 40dp but the design size is 32dp. Can you apply it?

* Tab height fixed at 40 dp.
* Margin between Tab and ViewPager was narrow and it was widened.

Please let me know if other approaches are good 🙇 

https://github.com/DroidKaigi/conference-app-2019/pull/342/commits/483af06f32e80dba020bb5def785ec63f3fa2411

device-2019-01-10-174959

**Comment by takahirom** [comment]
I thought it was impossible, but thanks for making a bit of a change. 🏆

**Pull request merged**